src/RA_Defs.h

```diff
@@ -300,46 +300,46 @@ const int SERVER_PING_DURATION = 2 * 60;
 
 #define RA_LOG RADebugLog
 
-#ifdef _DEBUG
+#if _DEBUG
 #ifndef RA_UTEST
 #undef ASSERT
 #define ASSERT( x ) assert( x )
 #else
 #undef ASSERT
 #define ASSERT( x ) {}
-#endif
-#else
-#undef ASSERT
-#define ASSERT( x ) {}
```

Comment:
I currently get around 200 build errors in RELEASE mode. About half of them are "ASSERT is undefined"

Reply:
the others fall into two categories:

not including in files that use unique_ptr
unit test code related to latest changes.

Reply:
I need to take a break for awhile after this. That is so strange. Don't know why these review comments aren't showing up when they should.

Well the ASSERT problem did exist, but I think the <memory> problem was a false error, I didn't get an error.

Reply:
this is what I get without manually adding the missing `#include <memory>`s:
```
1>e:\source\raintegration\src\ra_core.cpp(1555): error C2039: 'unique_ptr': is not a member of 'std'
```
```
1>e:\source\raintegration\src\ra_httpthread.cpp(343): error C2039: 'make_unique': is not a member of 'std'
```
Looks like it's just those two files.